docs/adguard-home/getting-started.md

```diff
@@ -0,0 +1,256 @@
+
+## Updating {#update}
+
+![An example of an update notification](https://cdn.adtidy.org/content/kb/dns/adguard-home/updatenotification.jpg)
+
+When a new version is released, AdGuard Home’s UI shows a notification message and the *Update now* button. Click this button, and AdGuard Home will be automatically updated to the latest version. Your current AdGuard Home executable file is saved inside the `backup` directory along with the current configuration file, so you can revert the changes, if necessary.
+
+### Manual update {#manual-update}
+
+In case the button isn’t shown or an automatic update has failed, you can update manually. We have a [detailed guide on manual updates][mupd], but in short:
+
+1. Download the new AdGuard Home package.
+
+1. Extract it to a temporary directory.
+
+1. Replace the old AdGuard Home executable file with the new one.
+
+1. Restart AdGuard Home.
+
+[mupd]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#manual-update
+
+### Docker, Home Assistant, and Snapcraft updates
+
+Auto-updates for Docker, Hass.io/Home Assistant, and Snapcraft installations are disabled. Update the image instead.
+
+### Command-line update
+
+To update AdGuard Home package without the need to use Web API run:
+
+```sh
+./AdGuardHome --update
+```
+
+## Configuring devices {#configure-devices}
+
+### Router
+
+This setup will automatically cover all devices connected to your home router, and you won’t need to configure each of them manually.
+
+1. Open the preferences for your router. Usually, you can access it from your browser via a URL, such as <http://192.168.0.1/> or <http://192.168.1.1/>. You may be prompted to enter a password. If you don’t remember it, you can often reset the password by pressing a button on the router itself, but be aware that if this procedure is chosen, you will probably lose the entire router configuration. If your router requires an app to set it up, please install the app on your phone or PC and use it to access the router’s settings.
+
+1. Find the DHCP/DNS settings. Look for the DNS letters next to a field that allows two or three sets of numbers, each divided into four groups of one to three digits.
+
+1. Enter your AdGuard Home server addresses there.
+
+1. On some router types, a custom DNS server cannot be set up. In that case, setting up AdGuard Home as a DHCP server may help. Otherwise, you should consult your router manual to learn how to customize DNS servers on your specific router model.
+
+### Windows
+
+1. Open *Control Panel* from the Start menu or Windows search.
+
+1. Go to *Network and Internet* and then to *Network and Sharing Center.*
+
+1. On the left side of the screen, find the *Change adapter settings* button and click it.
+
+1. Select your active connection, right-click it and choose *Properties.*
+
+1. Find *Internet Protocol Version 4 (TCP/IPv4)* (or, for IPv6, *Internet Protocol Version 6 (TCP/IPv6)*) in the list, select it, and then click Properties again.
```

Comment:
`Properties` should probably be wrapped into stars `*` for italic, just like in the previous list item